torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1341,27 +1343,123 @@ Tensor computeConv2d(
   return Tensor(ResultBuf.node(), s);
 }
 
-Tensor tensorexpr::computeOperandValue(
-    c10::Symbol op,
+Tensor computeReshape(
     const std::vector<ArgValue>& inputs,
     const std::vector<ExprHandle>& outputShape,
     const c10::optional<ScalarType>& outputType,
     at::Device device) {
-  switch (op) {
-    case aten::add: {
-      auto add_lambda = [](const ExprHandle& lhs, const ExprHandle& rhs) {
-        return boolToInteger(lhs) + boolToInteger(rhs);
-      };
-      TORCH_INTERNAL_ASSERT(
-          inputs.size() == 2 || inputs.size() == 3,
-          buildErrorMessage("Invalid number of input operands"));
-      return (inputs.size() > 2)
-          ? computeTwoOperandWithAlpha(
-                "aten_add", inputs, outputShape, outputType, add_lambda)
-          : computeTwoOperand(
-                "aten_add", inputs, outputShape, outputType, add_lambda);
-    } break;
-    case aten::sub: {
+  auto A = c10::get<BufHandle>(inputs[0]);
```

Comment:
I'm assuming this function and the ones below (`computeTranspose` and `computeExpand`) have just been refactored from below. So, not checking them in detail.